README.md

```diff
@@ -60,60 +60,12 @@ Configuration example in `mysite/_config/spamprotection.yml`
 
 Then once you have setup this config you will need to include the spam protector field as per the instructions on the [spamprotection](https://github.com/silverstripe/silverstripe-spamprotection) page.
 
-
-## Custom Theming
```

Comment:
Ah forget it, they've put the old v1 docs on the same nav as the v2 docs ... functionality doesn't exist in v2 any more.